roles/gdm/tasks/main.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Install GNOME Display Manager (GDM)
+  pacman: name=gdm state=present
+
+- name: Enable GDM
+  service: name=gdm.service enabled=yes
+
+- name: Creating DConf configuratioin profile
+  copy: src=profile/gdm dest=/etc/dconf/profile/gdm
+
+- name: Creating gdm.d directory for DConf
+  file: path=/etc/dconf/db/gdm.d state=directory owner=root group=root mode=0644
+  when: gdm.logo_path or
+        (gdm.font_scaling_factor > 1) or
+        gdm.sound or
+        gdm.touchpad.tap_to_click or
+        gdm.accessibility_menu
+
+- name: Log-in screen logo
+  template: src=gdm.d/02-logo.j2 dest=/etc/dconf/db/gdm.d/02-logo
```

Comment:
Rather than putting all of these config options in separate files, it may be simpler to just push out a single `00-spark` file (or something with a similar name) from a template. You could put all the conditionals in there, such as:

```
...
[org/gnome/desktop/sound]
event-sounds='{% if gdm.sound %}true{% else %}false{% endif %}'

[org/gnome/desktop/peripherals/touchpad]
tap-to-click='{% if gdm.touchpad.tap_to_click %}true{% else %}false{% endif %}'
...
```

That simplifies the role a bit, and has the added benefit of allowing the user to change their options via Ansible. Currently, if `gdm.touchpad.tap_to_click` is `True`, the feature will be enabled, but if the user then changes the variable to `False` and runs the role again, that task will just be skipped without disabling the feature.